kong/db/init.lua

```diff
@@ -81,6 +83,7 @@ function DB.new(kong_config, strategy)
     daos       = daos,       -- each of those has the connector singleton
     strategies = strategies,
     connector  = connector,
+    strategy   = strategy,
```

Comment:
I had the exact same change done in my codebase here as I work on the Plugins entity. :) We really need to start making smaller PRs with this little "minor tweaks" as we work through major changes!